website/docs/integrator-section/transferable-record/issuing-transferable-record.md

```diff
@@ -11,24 +11,27 @@ In this final step, we will create the transferable record and initialize the fi
 You will require the following:
 
 - token registry address
-- private key to token registry (key.txt)
+- private key to token registry (`key.txt`)
 - merkle root of the wrapped transferable record
+- beneficiary and holder wallet addresses
 
-## Issuing Transferable Records
+## Issuing the transferable record
 
-In the example, we will use 0x6FFeD6E6591b808130a9b248fEA32101b5220eca for as beneficiary and holder. You will need to replace this value with a wallet address you control to be able to perform different actions on the transferable records later.
+In the example, we will use 0x6FFeD6E6591b808130a9b248fEA32101b5220eca as beneficiary and holder. You will need to replace this value with a wallet address you control, to be able to perform different actions on the transferable records later.
 
-To issue the transferable record, simply run the following command:
+To issue the transferable record, replace the following values:
+
+- `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with your token registry contract address
+- `0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea` with your merkle root (with a `0x` prefix)
+- `0x6FFeD6E6591b808130a9b248fEA32101b5220eca` with your beneficiary and holder addresses
+
+Then run the command:
 
 ```sh
 open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --tokenId 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea --beneficiary 0x6FFeD6E6591b808130a9b248fEA32101b5220eca --holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca -n sepolia -f key.txt
 ```
 
-Note to replace the following values:
-
-- `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with your token registry contract address
-- `0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea` with your merkle root (with a `0x` prefix)
-- `0x6FFeD6E6591b808130a9b248fEA32101b5220eca` with your beneficiary and holder addresses
+>**Note:** In this tutorial, we use the same beneficiary and holder address for demonstration purpose. You can specify different wallet addresses for the beneficiary and holder if they are two different entities.
```

Comment:
```suggestion
>**Note:** In this tutorial, we use the same beneficiary and holder addresses for demonstration purposes. You can specify different wallet addresses for the beneficiary and holder if they are two different entities.
```

Reply:
"purpose" is changed to "purposes".
For the same "address", I guess it should stay as the singular form instead of plural, 
because there's only one address